tests/unit/customizations/test_commands.py

```diff
@@ -136,6 +142,25 @@ def test_expected_events_are_emitted_in_order(self):
             'before-building-argument-table-parser.s3.foo'
         ])
 
+    def test_expected_events_are_emitted_in_order_with_event_name(self):
```

Comment:
We should also add a functional test specficially for sso to make sure we are not adding to the `configure sso` command table. Oddly enough a command like this:
```
aws configure sso logout help
```
currently works in the CLI and it should not. Maybe we can add a functional test to make sure `aws configure sso logout` is an invalid command?